integrations/airtable/src/setup/handler.ts

```diff
@@ -0,0 +1,11 @@
+import type { Handler } from '../misc/types'
+
+class NotImplementedError extends Error {
+  constructor() {
+    super('Not implemented')
+  }
+}
+
+export const handler: Handler = async () => {
```

Comment:
**chore:** There's no need to throw an error here. Just an empty handler will do. Also, since it's empty please move it to `integrations/airtable/src/index.ts`; there's no need for a dedicated file.